### PR TITLE
Add rocky9 to CI

### PR DESCRIPTION
**Pull request by zhrt123**
DEV pipeline: https://dev2.ci.gpdb.pivotal.io/teams/gp-extensions/pipelines/dev.diskquota.add-rocky9.zhrt

### CODE REVIEW

File: concourse/pipeline/job_def.lib.yml [hunk 1]
```diff
@@ -228,11 +254,31 @@ plan:
...
     resource: #@ conf["res_diskquota_bin"]
+#@   end
 - #@ _build_task(conf)
 - #@ _test_task(conf)
 - put: #@ conf["res_intermediates_bin"]
   params:
     file: diskquota_artifacts/diskquota.tar.gz
 #@ end
+
+#@ def get_groups(confs):
```
Comment (liuxueyang) [inline concourse/pipeline/job_def.lib.yml:268]: There is a bug in this function. Use `get_groups` in base.lib.yml instead.
For example https://github.com/pivotal/gp-extensions-ci/blob/main/projects/advanced-password-check/pr.yml#L23

Comment (liuxueyang) [reply concourse/pipeline/job_def.lib.yml:268]: If diskquota has not been migrated to the monorepo. I think the only way is to copy the function from https://github.com/pivotal/gp-extensions-ci/blob/main/common/base.lib.yml#L65 manually here.

Comment (zhrt123) [reply concourse/pipeline/job_def.lib.yml:268]: hmm, I will fix it.

**Comment by zhrt123** [comment]
The other PR has done the same thing, close this PR.

**Pull request closed without merging**